packages/forms-web-app/src/controllers/projects/examination-timetable.js

```diff
@@ -41,6 +41,8 @@ const eventSubmitButtonActive = (timetable) => {
 	);
 };
 
+const displayEvents = (dateOfNonAcceptance) => new Date(dateOfNonAcceptance) >= getDate();
```

Comment:
Is this correct - we want to only view timetable when current date is after the dateOfNonAcceptance (timetable published date)